quickstack/manifests/neutron/compute.pp

```diff
@@ -188,7 +188,7 @@
     $local_ip = find_ip("$ovs_tunnel_network","$ovs_tunnel_iface","")
     class { '::neutron::agents::ml2::ovs':
       bridge_uplinks   => $ovs_bridge_uplinks,
-      bridge_mappings  => $ovs_bridge_mappings,
+      #bridge_mappings  => $ovs_bridge_mappings,
```

Comment:
Yes, I might have to puppetize the step of how to bring down br-ex once everything is configured. I have to look into puppetizing that part, need to figure out the right way.